scripts/test_screeps_rl_live_dashboard.py

```
@@ -1211,6 +1256,75 @@ def test_live_acceptance_passes_running_service(self) -> None:
+                auto_refresh_seconds=60,
+            )
+            try:
+                server = live.make_server("127.0.0.1", 0, config)
+            except OSError as error:
+                self.skipTest(f"socket creation is unavailable in this sandbox: {error}")
+            server.record_refresh(initial_refresh, refreshed_at="2026-05-18T10:09:00Z")
+
+            def blocked_refresh_metrics(
+                db_path_arg: Path,
+                artifact_root_arg: Path,
+                paths: Any = None,
```

Ruff (0.15.13)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `paths`

(ANN401)